components/tidb_query/src/rpn_expr/impl_time.rs

```diff
@@ -128,13 +128,37 @@ pub fn year(ctx: &mut EvalContext, t: &Option<DateTime>) -> Result<Option<Int>>
     Ok(Some(Int::from(t.year())))
 }
 
+#[rpn_fn(capture = [ctx])]
+#[inline]
+pub fn month_name(ctx: &mut EvalContext, t: &Option<DateTime>) -> Result<Option<Bytes>> {
+    match t {
+        Some(t) => {
+            let month = t.month() as usize;
+            if t.is_zero() && ctx.cfg.sql_mode.contains(SqlMode::NO_ZERO_DATE) {
+                ctx.handle_invalid_time_error(Error::incorrect_datetime_value(t))
+                    .map(|_| Ok(None))?
+            } else if month == 0 {
+                Ok(None)
+            } else {
+                use crate::codec::mysql::time::MONTH_NAMES;
+                Ok(Some(MONTH_NAMES[month - 1].to_string().into_bytes()))
```

Comment:
Would you please add a trait `MonthExtension` to `components/tidb_query/src/codec/mysql/time/extension.rs`? It will be more elegant to write this code just like: `Ok(Some(t.month_name()))`

Reply:
ok